content/03.use-cases-a.md

```diff
@@ -1,30 +1,42 @@
-## Thirteen practical ways GitHub can accelerate research in ecology and evolution
+## Twelve practical ways GitHub can accelerate research in ecology and evolution
 
 ### Storing and sharing research compendia
 
 <!--*contributors to this section: Dylan Gomes, Emma Hudgins, Pedro Braga, Katherine Hébert* -->
-An EEB research compendium includes all computational materials related to research production, including data, code for analyses and protocols.
-Having copies of these files safely stored is essential to protect against accidental modifications or deletions.
-Many researchers begin using GitHub to store (or backup) their research compendium [@doi:10.1080/00031305.2017.1375986] to a centralized, readily-available remote server (see [Box 1](#definitions)).
-A centralized research compendium stored in a version-controlled repository has the advantages of facilitating collaboration, integrating data and code archiving services, allowing file versions to be accessed and restored, and contributes to open science (see sections below).
-While some EEB researchers may prefer to store their code in Google Drive or DropBox, the process of sharing code through these services becomes cumbersome when collaborators have to download and re-upload edited code.
+An EEB research compendium includes all computational materials related to research production, including data, code for analyses and protocols. 
+Safely storing these files is essential to protect against accidental modifications or deletions.
+Many researchers begin using GitHub to backup their research compendium [@doi:10.1080/00031305.2017.1375986] into a centralized, readily-available remote server (see [Box 1](#definitions)).
+A centralized research compendium stored in a version-controlled repository has the advantages of facilitating collaboration, integrating data and code archiving, allowing file versions to be accessed and restored, and further contributing to open science practices (@doi:10.1162/99608f92.9497f68e).
+Every change made to files in a version-controlled repository accompain a description of modifications and are authored ([Box 1](#definitions)).
+Later, the entire history of commits and their commit messages are viewable and can be audited similar to a physical laboratory notebook [@doi:10.1186/1751-0473-8-7].
+While some EEB researchers prefer to store their code in alternative cloud services, such as Google Drive, OneDrive and DropBox, the process of sharing code through these services becomes cumbersome when collaborators have to download and re-upload edited code.
```

Comment:
This one is not clear to me as well. In my experience, the ability to automatically sync these files from the computer is actually great. There is no requirement to commit and push, these services just sync whatever changes are made, they keep a version history that one can even name, and sometimes multiple users can simultaneously collaborate in a file. While detailing what cumbersome is necessary, I am wondering whether this sentence is essential for this part.

Reply:
True, although dropbox and one drive don't play nicely with RStudio projects, I think (maybe this has been fixed).  But you're right that there's probably no need to download and re-upload.  I've never really tried using these services for anything other than collaborative editing of word/google docs.  Let's just remove it?
```suggestion

```

Reply:
I'm silly! Now I understand what you meant by the suggested commits.

Reply:
Alright!